test/functional/test_framework/messages.py

```diff
@@ -0,0 +1,1437 @@
+
```

Comment:
Should not have an empty line here

Reply:
https://github.com/dashpay/dash/pull/3372/commits/cb91d15055edc530213ca75cb145fd9220c10e82